tanstack-table/src/main/scala/lucuma/react/table/facade/ColumnDefJS.scala

```diff
@@ -28,9 +28,9 @@ trait ColumnDefJS[T, A] extends js.Object:
   var enableHiding: js.UndefOr[Boolean] = js.undefined
 
   // Sorting
-  var enableMultiSort: js.UndefOr[Boolean]        = js.undefined
-  var enableSorting: js.UndefOr[Boolean]          = js.undefined
-  var invertSorting: js.UndefOr[Boolean]          = js.undefined
-  var sortDescFirst: js.UndefOr[Boolean]          = js.undefined
-  var sortUndefined: js.UndefOr[false | -1 | 1]   = js.undefined
-  var sortingFn: js.UndefOr[raw.mod.SortingFn[A]] = js.undefined
+  var enableMultiSort: js.UndefOr[Boolean]                 = js.undefined
+  var enableSorting: js.UndefOr[Boolean]                   = js.undefined
+  var invertSorting: js.UndefOr[Boolean]                   = js.undefined
+  var sortDescFirst: js.UndefOr[Boolean]                   = js.undefined
+  // var sortUndefined: js.UndefOr[false | -1 | 1]   = js.undefined
```

Comment:
Commented code

Reply:
This is pending implementation in the facade. Will address in future PR.